kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala

```diff
@@ -44,6 +44,16 @@ object TypelevelKernelPlugin extends AutoPlugin {
   override def buildSettings =
     addCommandAlias("tlReleaseLocal", mkCommand(List("reload", "project /", "+publishLocal")))
 
+  override def projectSettings = Seq(
+    skip := {
+      skip.value || {
+        val cross = crossScalaVersions.value
+        val ver = (LocalRootProject / scalaVersion).value
+        !cross.contains(ver)
+      }
+    }
+  )
+
```

Comment:
I'd say this is a good upstream candidate too, but maybe too many builds are depending on it not being this way. :shrug:

Reply:
I was wondering about this. Wouldn't this break the trick we use to back-publish kind projector?

Reply:
Oh, poo.  I think it would.  We could counterhack that, but yeah.  I don't think this would fly upstream.